src/main/java/com/xceptance/neodymium/util/SelenideAddons.java

```diff
@@ -160,7 +160,8 @@ public ElementsCollection get()
             }
             catch (final Throwable t)
             {
-                if (isThrowableCausedBy(t, StaleElementReferenceException.class))
+                if (isThrowableCausedBy(t, StaleElementReferenceException.class)
```

Comment:
Could you refacture a private function out of the analysis of the Throwable and reuse  on line 227 to reduce the amount of duplicated code

Reply:
Done by reusing `$safe(final Supplier<SelenideElement> code)` method